[Feature]  Adds `userAgent` string to Freshdesk ticket

mnigh opened the pull request:
🤖 Resolves #12072.

## 👋 Introduction

This PR adds the [userAgent](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent) string to a new support ticket via to the `SupportForm` component.

## 📸 Screenshot

<img width="1066" alt="Screen Shot 2024-12-03 at 16 21 59" src="https://github.com/user-attachments/assets/d133eb06-2dbe-44e2-a472-134f70825afa">

## 🧪 Testing

> [!NOTE]  
> To test creating a support ticket, you will need to access to and [credentials for Freshdesk](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/documentation/freshdesk.md).

1. `make refresh-api; pnpm build`
2. Navigate to http://localhost:8000/en/support
3. Fill out the support form and submit
5. Verify User Agent field in newly created Freshdesk ticket has the appropriate `userAgent` string value

petertgiles commented:
One of the AC is "Last link viewed".  Was that fixed in this PR?  It seems to be all about UA.

mnigh commented:
> One of the AC is "Last link viewed". Was that fixed in this PR? It seems to be all about UA.

There was nothing to fix. The **Page URL** field that is sent through as a custom field to the Freshdesk ticket is still working as it did before and is rendered in the ticket. I think it was mistakenly added to the acceptance criteria as it is one of the items rendered in the tickets from the Freshdesk support portal.

mnigh commented:
> > One of the AC is "Last link viewed". Was that fixed in this PR? It seems to be all about UA.
> 
> There was nothing to fix. The **Page URL** field that is sent through as a custom field to the Freshdesk ticket is still working as it did before and is rendered in the ticket. I think it was mistakenly added to the acceptance criteria as it is one of the items rendered in the tickets from the Freshdesk support portal.

@petertgiles For clarity, I've rewritten [the issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12072) with new acceptance criteria and changed it from a bug to a feature since that is what the issue really is. Please let me know if you have any further questions.

esizer reviewed (approved):
LGTM! :+1: It even knows I am on X11 :stuck_out_tongue: Maybe I should test wayland :eyes: 

![2024-12-04_11-46](https://github.com/user-attachments/assets/753ed471-93b0-40a7-a305-b2d039e4ac3f)
